ci/README.md

```diff
@@ -32,27 +46,30 @@ This directory contains scripts used for the development of code-server.
 ## build
 
 This directory contains the scripts used to build code-server.
+You can disable minification by setting `MINIFY=`.
 
+- [./lib.sh](./lib.sh)
+  - Contains code duplicated across these scripts.
 - [./build/build-code-server.sh](./build/build-code-server.sh) (`yarn build`)
   - Builds code-server into ./out and bundles the frontend into ./dist.
 - [./build/build-vscode.sh](./build/build-vscode.sh) (`yarn build:vscode`)
   - Builds vscode into ./lib/vscode/out-vscode.
 - [./build/build-release.sh](./build/build-release.sh) (`yarn release`)
-  - Bundles the output of the above two scripts into a single node module at ./release.
-  - Will build a static release with node/node_modules into `./release-static`
-    if `STATIC=1` is set.
+  - Bundles the output of the above two scripts into a single node module at `./release`.
+- [./build/build-static-release.sh](./build/build-static-release.sh) (`yarn release:static`)
+  - Requires a release already built in `./release`.
+  - Will build a static release with node and node_modules into `./release-static`
 - [./build/clean.sh](./build/clean.sh) (`yarn clean`)
   - Removes all git ignored files like build artifacts.
   - Will also `git reset --hard lib/vscode`
   - Useful to do a clean build.
 - [./build/code-server.sh](./build/code-server.sh)
   - Copied into static releases to run code-server with the bundled node binary.
-- [./build/archive-static-release.sh](./build/archive-static-release.sh)
-  - Archives `./release-static` into a tar/zip for CI with the proper directory name scheme
 - [./build/test-release.sh](./build/test-static-release.sh)
   - Ensures code-server in the `./release-static` directory runs
-- [./build/build-static-pkgs.sh](./build/build-static-pkgs.sh) (`yarn pkg`)
-  - Uses [nfpm](https://github.com/goreleaser/nfpm) to generate .deb and .rpm from a static release
+- [./build/build-packages.sh](./build/build-static-pkgs.sh) (`yarn package`)
```

Comment:
The names on the left don't match the right. Looks like on the first one the right is correct and on the second the left is correct.